src/cli/main.cpp

```diff
@@ -417,6 +425,10 @@ int main(int argc, char* argv[]) {
     rb_provide("fiddle");
     rb_provide("fiddle.so");
 
+    Init_generator();
+    rb_provide("json/ext/generator");
+    rb_provide("json/ext/generator.so");
```

Comment:
Simple reorder alphabetically

Reply:
amazed these were able to slip by for so long.